### DISCUSSION TITLE
Documentation page is very heavy

### DISCUSSION THREAD

**Discussion by kuwabara74** [materialsproject/pymatgen#3554] [Q&A] [answered]
The documentation web page of [`pymatgen.io.vasp`](https://pymatgen.org/pymatgen.io.vasp.html) is slow to load. I suppose the description of "potcar_summary_stats" is very long.
Could you change the contents so that they do not appear on the web page or only part of the contents are shown?

**Comment by janosh** [comment, marked as answer]
Thanks for reporting @kuwabara74! Fix is being tracked in https://github.com/materialsproject/pymatgen/issues/3556.

**Comment by kuwabara74** [comment]
Thank you for your response, @janosh.
I appreciate that you and @esoteric-ephemera handled the issue of #3556 (the attr of potcar_summary_stats has been changed to a private type).

However, the documentation page of pymatgen.io.vasp still shows the potcar_summary_stats and is prolonged to load yet now.  

I tried to open the page with Google Chrome and Microsoft Edge. I also reloaded the page after deleting the cache data of my browsers. The situation is the same in all cases. So, the problem does not seem to have been solved.

**Reply by janosh** [reply]
[the docs should update on every commit](https://github.com/materialsproject/pymatgen/actions/runs/7553131457) but we also noticed it's still there. not sure why, didn't look into this further. might the next release will fix it

**Reply by kuwabara74** [reply]
Thank you for your reply. I will wait for the next revision and hopefully, the problem will be solved.
